src/details/ArborX_DetailsDistributor.hpp

```diff
@@ -124,9 +120,7 @@ class Distributor
     int comm_size;
     MPI_Comm_size(_comm, &comm_size);
 
-    _permute = Kokkos::View<int *, Kokkos::HostSpace>(
-        Kokkos::ViewAllocateWithoutInitializing("permute"),
-        destination_ranks.size());
+    reallocWithoutInitializing(_permute, destination_ranks.size());
```

Comment:
Do you reckon we should have add `_permute` to the member initializer list in the constructor rather than relying on the default constructor?
```cpp
, _permute{Kokkos::ViewAllocateWithoutInitializing("permute"), 0}`
```
I feel like we should provide a label.

Reply:
Sure, I am happy to add that.